presto-hive/src/main/java/com/facebook/presto/hive/HiveSplitSource.java

```diff
@@ -158,13 +158,13 @@ public static HiveSplitSource allAtOnce(
                     }
 
                     @Override
-                    public void finish()
+                    public void finishSplitLoading()
```

Comment:
`finishSplitLoading` implies that this method finished it. What it really does, it is called when split enumeration is finished.

All across the code we have this unspoken convetion of calling method / fields `noMore` (e.g.: noMorePages, noMoreSplits, noMoreDrivers). How about calling this method the same way: `noMoreSplits`